packages/syft/src/syft/service/dataset/dataset_service.py

```diff
@@ -52,7 +52,10 @@ def add(
         )
         if result.is_err():
             return SyftError(message=str(result.err()))
-        return SyftSuccess(message="Dataset Added")
+        return SyftSuccess(
+            message=f"Dataset uploaded to {context.node.name}."
+            f"To see datasets on this node use command `client.api.services.dataset.get_all()`"
```

Comment:
We can just call .datasets. `client.datasets`

Reply:
Ak thanks @shubham3121